site/assets/js/search.js

```diff
@@ -12,6 +12,14 @@
   var inputElement = document.getElementById('search-input')
   var siteDocsVersion = inputElement.getAttribute('data-docs-version')
 
+  document.addEventListener('keydown', function (event) {
+    if (event.key === '/') {
+      event.preventDefault()
+
+      inputElement.focus()
+    }
+  })
+
```

Comment:
Actually this code is not necessary anymore, because algolia uses [`autocompleteOptions`](https://docsearch.algolia.com/docs/behavior/#autocompleteoptions), now we can set the keycode value of `/` to handle this behaviour using the `keyboardShortcuts` option.

When I opened #29834 I forgot to check this in the `docsearch` documentation

Reply:
Adding this options to the `window.docsearch` should be do the work:
```js
window.docsearch({
  […],
  autocompleteOptions: {
    keyboardShortcuts: [111]
  }
})
```
Where `111` is the keycode value of `/` key